### PR TITLE
[Enhancement] Backend refactor

### PR DESCRIPTION
**Pull request by Marioneq4958**

### CODE REVIEW

File: backend/tests/endpoints/login.py [hunk 1]
```diff
@@ -17,15 +17,11 @@ def login_test(nick, password, host, symbol, ssl, fg):
...
+    headers = login.json()[0]["schools"][0]["headers"]
+    student = login.json()[0]["schools"][0]["students"][0]["cookies"]
+    school_id = login.json()[0]["schools"][0]["id"]
     status_check(login.status_code, login.json(), fg)
-    assert login.json()["symbol"] == "powiatwulkanowy"
-    assert login.json()["vulcan_cookies"]
-    try:
-        assert login.json()["host"] == "fakelog.cf"
-    except:
-        assert login.json()["host"] == "fakelog.tk"
```
Comment (Zaptyp) [inline backend/tests/endpoints/login.py:30]: Why this code be deleted?

Comment (Zaptyp) [reply backend/tests/endpoints/login.py:30]: Revert this change

**Review by Zaptyp — CHANGES_REQUESTED** [review]
Nice!

File: backend/tests/endpoints/login.py (hunk 1 above)
Comment (Marioneq4958) [reply backend/tests/endpoints/login.py:30]: Because now there isn't `host` in `login.json()`

**Review by Zaptyp — APPROVED** [review]

**Pull request merged**